### PR TITLE
Replace inline stylings with Material-UI Box attributes

### PR DESCRIPTION
**Pull request by amaaniqbal**
This PR replaces inline stylings present in `tag/` and `team/` directories inside `check-web/src/app/components/` with the corresponding Material-UI `Box` component attributes.

Fixes [#18](https://github.com/meedan/check/issues/18)

### CODE REVIEW

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Thanks @amaaniqbal for your contribution! We appreciate it! Please notice that it won't work if you don't import material-ui `Box` on those files! Please fix as needed so we are able to accept it! Thanks again for contributing!

**Comment by amaaniqbal** [comment]
Oops, missed it! Fixing it right away...

File: src/app/components/team/TeamBots.js [hunk 1]
```diff
@@ -188,157 +189,160 @@ class TeamBotsComponent extends Component {
     return (
       <ContentColumn style={{ maxWidth: 900 }}>
         { team.team_bot_installations.edges.length === 0 ?
-          <p style={{ paddingBottom: units(5), textAlign: 'center' }}>
+          <Box pb={5} display='flex' alignItems='center' justifyContent='center'>
```
Comment (amoedoamorim) [inline src/app/components/team/TeamBots.js:192]: Please use double quotes for string props/attributes.

`192:31  error  Unexpected usage of singlequote  jsx-quotes`

File: src/app/components/team/TeamReport.js [hunk 2]
```diff
@@ -85,88 +86,98 @@ class TeamReportComponent extends React.Component {
           onChange={this.handleChangeLanguage.bind(this)}
         />
         <Can permissions={team.permissions} permission="update Team">
-          <p style={{ marginTop: units(2), textAlign: 'end' }}>
-            <Button onClick={this.handleSubmit.bind(this)} primary>
-              <FormattedMessage id="teamReport.save" defaultMessage="Save" />
-            </Button>
-          </p>
+          <Box mt={2} display='flex' alignItems='end' justifyContent='end'>
```
Comment (amoedoamorim) [inline src/app/components/team/TeamReport.js:89]: Please use double quotes for string props/attributes.

`89:31  error  Unexpected usage of singlequote    jsx-quotes`

File: src/app/components/team/TeamTags.js [hunk 3]
```diff
@@ -307,95 +308,100 @@ class TeamTagsComponent extends Component {
 
   tagsList(list) {
     return (
-      <List
-        style={{
-          maxHeight: 400,
-          overflowY: 'auto',
-          overflowX: 'hidden',
-          padding: units(2),
```
Comment (amoedoamorim) [inline src/app/components/team/TeamTags.js:315]: Please remove units from import statement.

`27:10  error  'units' is defined but never used  no-unused-vars`

File: src/app/components/team/TeamTags.js [hunk 4]
```diff
@@ -466,48 +472,55 @@ class TeamTagsComponent extends Component {
...
-              </div>
-            </Can>
-          </CardContent>
-        </Card>
+        <Box mt={2}>
+          <Card>
+            <Box p={0}>
+              <CardContent>
+                { tag_texts.length === 0 ?
+                  <Box pb={5} display='flex' alignItems='center'>
```
Comment (amoedoamorim) [inline src/app/components/team/TeamTags.js:489]: Please use single quotes for string props/attributes.

`489:39  error  Unexpected usage of singlequote    jsx-quotes`

File: src/app/components/team/TeamReport.js [hunk 5]
```diff
@@ -85,88 +86,98 @@ class TeamReportComponent extends React.Component {
           onChange={this.handleChangeLanguage.bind(this)}
         />
         <Can permissions={team.permissions} permission="update Team">
-          <p style={{ marginTop: units(2), textAlign: 'end' }}>
```
Comment (amoedoamorim) [inline src/app/components/team/TeamReport.js:88]: Please remove units from import statement.

12:25 error 'units' is defined but never used no-unused-vars

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Hello @amaaniqbal ! Thank you so much for your interest in contributing! Please notice that some coding standards need to be met and we require a few changes to accept this PR. I kindly ask you to review your PR and apply these changes. Mainly the string props should be enclosed in double quotes, and unused imports should be removed.

**Comment by amaaniqbal** [comment]
@amoedoamorim I did the suggested changes. Please have a look!

**Comment by amoedoamorim** [comment]
Thanks @amaaniqbal for your contribution! We appreciate it! There are a couple of conflicts in your PR in relation to current `develop` branch. Would you solve these so we can move forward with merging your code, please?

**Comment by amaaniqbal** [comment]
Fixed the conflicts @amoedoamorim!

**Comment by amaaniqbal** [comment]
@amoedoamorim any more changes required?

**Comment by amoedoamorim** [comment]
Hello @amaaniqbal ! Thanks for your contribution! We appreciate it! Your PR looks good in general, but there are some conflicts resulting from merges of PRs from others. Please solve those so we can go ahead and merge yours!

**Comment by amaaniqbal** [comment]
Fixed the merge conflicts again @amoedoamorim!

**Comment by alexmeedan** [comment]
Hello @amaaniqbal, I was away for the past weeks and was unable to merge your PR. Would you please fix this conflict so I can merge your PR? Thank you!

**Comment by amaaniqbal** [comment]
Hello @alexmeedan, this PR was up since Hacketoberfest and since then I am just resolving merge conflicts. I resolved them once again, but please review this before merging any other PRs this time if possible. Also, please consider this as a part of Hacktoberfest.

PS: `src/app/components/team/TeamProjects.js` was deleted by someone which resulted in merge conflicts, so I added it back. Let me know if it should be deleted, and I will remove the same.

**Comment by alexmeedan** [comment]
Yes, I'll merge as soon as it's fixed! I couldn't do it before because your previous commit was sent in while I was away. It will still count for Hacktoberfest and eligible for Meedan swag. I apologize for the inconvenience. The file `TeamProjects.js` is not in use anymore and should be deleted.

**Comment by amaaniqbal** [comment]
Thanks a lot for the positive response @alexmeedan! I removed the unused file `TeamProjects.js`. You can review the PR now.

**Comment by amoedoamorim** [comment]
Hello @amaaniqbal . I did some fixing, merging and cleaning up myself and merged your contribution on another branch. Thanks again for participating in Hacktoberfest with Meedan! We appreciate your contribution!

**Comment by amaaniqbal** [comment]
Thanks a lot for merging my changes @amoedoamorim! I can see them in the PR over [here](https://github.com/meedan/check-web/pull/768). May I know what made my previous commits fail Travis build and your commit passed? 
PS: I am just curious to know this.

**Pull request closed without merging**